NOTICE.txt

```diff
@@ -145,70 +149,6 @@ LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
 OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
 SOFTWARE.
 
----
```

Comment:
Is it safe to remove these notices?

Reply:
Yes. If you search for these licenses in the code base you'll notice that they no longer exist (outside of node-modules; the NOTICE just encapsulates code that was copied/modified in some way and re-distributed in the OSD repo). The reason they're being removed now is because there wasn't an automated process to update the NOTICE file in the CI.

Reply:
good to know